step_start_instance.go

```diff
@@ -66,6 +67,14 @@ func (s *stepStartInstance) Run(state multistep.StateBag) multistep.StepAction {
 		"image_name":  instance.ImageName(),
 	}).Info("started instance")
 
+	libhoney.SendNow(map[string]interface{}{
+		"provider":    s.provider,
+		"job_id":      buildJob.Payload().Job.ID,
+		"boot_time":   time.Since(startTime),
+		"instance_id": instance.ID(),
+		"image_name":  instance.ImageName(),
+	})
```

Comment:
perhaps we should also add a field for the name of the event, e.g. `"event": "started instance"` or such? not super important for a first test, but if we want to add more events we'll want to differentiate.